usr/share/lib/img_proof/tests/SLES/conftest.py

```diff
@@ -552,6 +552,28 @@
     repo.replace('SP5', 'SP6') for repo in SLE_15_SP5_X86_64_MODULES
 ]
 
+BASE_15_SP7 = [
+    repo.replace('SP6', 'SP7') for repo in BASE_15_SP5
+]
+BASE_15_SP7_SAP = [
+    repo.replace('SP6', 'SP7') for repo in BASE_15_SP5_SAP
+]
+BASE_15_SP7_HPC = [
+    repo.replace('SP6', 'SP7') for repo in BASE_15_SP5_HPC
+]
+SLE_15_SP7_X86_64_MODULES = [
+    repo.replace('SP6', 'SP7') for repo in SLE_15_SP5_X86_64_MODULES
+]
+
+SYSTEMS_MGMT_15_MODULE = [
+    'SLE-Module-Systems-Management-15-SP7-Pool',
+    'SLE-Module-Systems-Management-15-SP7-Updates'
+]
+
+BASE_15_SP7 += SYSTEMS_MGMT_15_MODULE
+BASE_15_SP7_SAP += SYSTEMS_MGMT_15_MODULE
+BASE_15_SP7_HPC += SYSTEMS_MGMT_15_MODULE
+
 SLES_REPOS = {
```

Comment:
We'd require to add entries for `15-SP7` in this dictionary, right?

Reply:
Yes, duh. Thanks for catching this. Fixed